student.rb

```diff
@@ -1,14 +1,7 @@
-require_relative 'person'
+class Student
```

Comment:
- [ ] Please make sure that the `Student` class inherits from the `Person` class because the central point for the `Person` is being the parent class for whatever we want to create later as students, teachers, and soever.  The `Person` class has the main methods that are being inherited by the `Student` class. For example, the renting process, could be for the teacher and could be for the student, but both are person, does that make sense?